src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,129 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    public static final int INITIAL_ARRAY_SIZE = 10;
+    public static final int RESIZE_NUMBER = 5;
```

Comment:
5 is not a correct value here, because each time you grow the array, you will do +5, and you should multiply by 1.5